platform/uwp/export/export.cpp

```diff
@@ -1110,11 +1117,20 @@ class EditorExportPlatformUWP : public EditorExportPlatform {
 		}
 
 		valid = dvalid || rvalid;
+
+#ifdef MODULE_MONO_ENABLED
+		// If this is a Mono build, provide a custom error so that users are not confused.
+		// We don't bypass the whole logic to check templates because third-parties might have
+		// Mono-enabled UWP builds using this path.
+		r_missing_templates = false; // Don't warn about those.
+		r_error = TTR("Godot's Mono version does not support the UWP platform. Use the standard build (no C# support) if you wish to target UWP.");
```

Comment:
Probably `valid` should forced to `false` here, in case some template files were placed into the mono template folder manually.

Reply:
No that's intentional as I mentioned in the comment:
```
// We don't bypass the whole logic to check templates because third-parties might have
// Mono-enabled UWP builds using this path.
```

The idea is that if you do provide templates which you got from a third-party and which would work, then you can use them. But if you don't, then it will just inform that the official Godot Mono doesn't support UWP.